### DISCUSSION TITLE
Is there anyway to use lazer skin element instead of using classic fallback one right now?

### DISCUSSION THREAD

**Discussion by IkaWaAyuMu** [ppy/osu#13478] [Q&A] [answered]
Is there anyway to use lazer skin element instead of using classic one right now? Such as to remove 100% judgement feedback, change only hitsound, import `MainHUDComponents.json` from someone's export etc.

**Comment by peppy** [comment, marked as answer]
So you're expecting to be able to replace legacy skin elements while also.... using the default lazer skin?

No, this is not supported yet.

**Reply by IkaWaAyuMu** [reply]
> No, this is not supported yet.

You mean, it (will) be selectable or just whole new element?

*(I'm kinda sad now that just import `MainHUDComponents.json` alone with `skin.ini` doesn't use lazer element yet)*

**Reply by peppy** [reply]
I don't know what your question means, but here's some food for thought:

You're asking for the ability to use legacy skin elements and provide resources for them but *not* use classic fallbacks. This is probably what we want in *all* cases, for anything that isn't a `GetDrawable` request. Which is to say something I've brought up previously - users should probably be able to still consume legacy components in a lazer-first skin (and there should always be a legacy provider for this case).

The differentiation should be what the default HUD layout and default drawable components are provided. This could already be done for the HUD, but you are also asking for it in other places we haven't yet implemented the skin layout system.

This isn't something we're just going to hack in place, but an issue which needs to be well thought out, as it will likely be staying around for some time in the future (and used for new lazer-first skins in some instances).
